examples/kubernetes/access_points/README.md

```diff
@@ -2,7 +2,7 @@
 Like [volume path mounts](../volume_path), mounting [EFS access points](https://docs.aws.amazon.com/efs/latest/ug/efs-access-points.html) allows you to expose separate data stores with independent ownership and permissions from a single EFS volume.
 In this case, the separation is managed on the EFS side rather than the kubernetes side.
 
-**Note**: Because access point mounts require TLS, this is not supported in driver versions at or before `0.3`.
+**Note**: Because access point mounts require TLS, this is not supported in driver versions at or before `0.3.0`.
```

Comment:
this seems wrong? Isn't it possible to use both tls and accesspoint mount options even on 0.3.0? If you have time to verify this statement would appreciate it ...